src/video.js

```diff
@@ -17,6 +58,28 @@ export function fillVideoDefaults(adUnit) {
   }
 }
 
+/**
+ * validateOrtbVideoFields mutates the `videoParams` object by removing invalid ortb properties.
+ * Other properties are ignored and kept as is.
+ *
+ * @param {object} videoParams
+ * @returns {void}
+ */
+export function validateOrtbVideoFields(videoParams) {
+  if (videoParams != null) {
+    Object.entries(videoParams)
+      .forEach(([key, value]) => {
+        if (ORTB_VIDEO_PARAMS.has(key)) {
+          const valid = ORTB_VIDEO_PARAMS.get(key).validate(value);
+          if (!valid) {
+            delete videoParams[key];
+            logWarn(`Invalid value for mediaTypes.video.${key} ORTB property. The property has been removed.`);
```

Comment:
Is this enough info to track down the issue - should we log the offending adUnit?

Reply:
This way, I should update the function to `validateOrtbVideoFields(adUnit);` would it sound better?

In 1st attempt I would like to keep the function scoped on the videoParams object, but perhaps it makes no sense.

Reply:
I tend to agree that the scope should be as small as possible, but I'm not sure code aesthetics will convince someone with a complicated setup and a bunch of cryptic warning messages :) 

You could also do something like `validateOrtbVideoFields(videoParams, onInvalidParam)` to keep the scope limited, but let something else that has the adUnit do the logging.